test-infra/prow/cluster/sinker_deployment.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     spec:
       containers:
       - name: sinker
-        image: gcr.io/k8s-prow/sinker:0.21
+        image: gcr.io/k8s-prow/sinker:0.24
```

Comment:
Will we have to keep updating these images? Is there a release tag we could use instead?

Reply:
They don't have release tags yet.